### PR TITLE
feat(builds): Adds troubleshooting playbook for build cops

### PR DESCRIPTION
**Pull request by ttomsu**
Build cops gave feedback that they didn't know what to do when on rotation and a build failed. This (illustrated) guide is an attempt to aid newcomers on navigating the Jenkins instance.

cc @spinnaker/build-cops

### CODE REVIEW

**Review by ezimanyi — APPROVED** [review]

File: community/contributing/nightly-builds/index.md [hunk 1]
```diff
@@ -81,3 +81,54 @@ The following jobs assist in removing old artifacts created during the build pro
 * [Admin_DailyJanitor](https://builds.spinnaker.io/view/5%20Admin/job/Admin_DailyJanitor/){:target="\_blank"}
 * [Admin_AuditBoms](https://builds.spinnaker.io/view/5%20Admin/job/Admin_AuditBoms/){:target="\_blank"}
 * [Admin_DeleteObsoleteArtifacts](https://builds.spinnaker.io/view/5%20Admin/job/Admin_DeleteObsoleteArtifacts/){:target="\_blank"}
+
+## Troubleshooting Playbook
+
+Check whether the failure happened during the _build_ or the _test_ phase:
+
+1. Click the failing Flow
```
Comment (dorbin) [inline community/contributing/nightly-builds/index.md]:
```suggestion
1. Click the failing Flow.
```

File: community/contributing/nightly-builds/index.md [hunk 2]
```diff
@@ -81,3 +81,54 @@ The following jobs assist in removing old artifacts created during the build pro
...
+
+    ![](troubleshooting - base - 10 - flow.png)
+
+1. Click for the most recent failing build
```
Comment (dorbin) [inline community/contributing/nightly-builds/index.md]:
```suggestion
1. Click for the most recent failing build.
```

File: community/contributing/nightly-builds/index.md [hunk 3]
```diff
@@ -81,3 +81,54 @@ The following jobs assist in removing old artifacts created during the build pro
...
+
+    ![](troubleshooting - base - 20 - mostRecent.png)
+
+1. Click through to the failing phase
```
Comment (dorbin) [inline community/contributing/nightly-builds/index.md]:
```suggestion
1. Click through to the failing phase.
```

File: community/contributing/nightly-builds/index.md [hunk 4]
```diff
@@ -81,3 +81,54 @@ The following jobs assist in removing old artifacts created during the build pro
...
+    ![](troubleshooting - build - 20 - buildSteps.png)
+
+1. Frequently, the build error will be printed out directly to the Console Output, but sometimes this output can be hard to read. View the raw file directly using the Build Artifacts link from Step 1.
+
+    ![](troubleshooting - build - 30 - failedOutput.png)
+
+
+### Test Failures
+
+1. View the Test Results Overview
```
Comment (dorbin) [inline community/contributing/nightly-builds/index.md]:
```suggestion
1. View the Test Results Overview.
```

File: community/contributing/nightly-builds/index.md [hunk 5]
```diff
@@ -81,3 +81,54 @@ The following jobs assist in removing old artifacts created during the build pro
...
+
+    ![](troubleshooting - test - 10 - testResultsOverview.png)
+
+1. Identify the failing test
```
Comment (dorbin) [inline community/contributing/nightly-builds/index.md]:
```suggestion
1. Identify the failing test.
```

**Review by dorbin — APPROVED** [review]

File: community/contributing/nightly-builds/index.md (hunk 1 above)
Comment (ttomsu) [reply community/contributing/nightly-builds/index.md]: Done.

File: community/contributing/nightly-builds/index.md (hunk 2 above)
Comment (ttomsu) [reply community/contributing/nightly-builds/index.md]: Done.

File: community/contributing/nightly-builds/index.md (hunk 3 above)
Comment (ttomsu) [reply community/contributing/nightly-builds/index.md]: Done.

File: community/contributing/nightly-builds/index.md (hunk 4 above)
Comment (ttomsu) [reply community/contributing/nightly-builds/index.md]: Done.

File: community/contributing/nightly-builds/index.md (hunk 5 above)
Comment (ttomsu) [reply community/contributing/nightly-builds/index.md]: I did all the other ones, but I _REFUSE_ to do this one!

**Pull request merged**